policyengine_us/variables/gov/states/dc/doee/liheap/dc_liheap_housing_type.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class DCLIHEAPHousingType(Enum):
+    SINGLE_FAMILY = "Single Family"
+    MULTI_FAMILY = "Multi Family"
+
+
+class dc_liheap_housing_type(Variable):
+    value_type = Enum
+    entity = SPMUnit
+    possible_values = DCLIHEAPHousingType
+    default_value = DCLIHEAPHousingType.SINGLE_FAMILY
```

Comment:
Should we default to multi family? Assuming more people live in apartments than houses